spec/lib/vmdb/loggers_spec.rb

```diff
@@ -10,12 +10,144 @@ def in_container_env(example)
     ENV['CONTAINER'] = old_env
   end
 
+  shared_examples "has basic logging functionality" do |logger|
+    subject       { logger }
+    let(:logger1) { subject }
+    let(:logger2) { $container_log }
+
+    before do
+      allow($container_log.logdev).to receive(:write)
+    end
+    before(:all) do
+      unless ENV['CONTAINER']
+        logger.extend(ActiveSupport::Logger.broadcast($container_log))
+      end
+    end
+
+    it "responds to #<<" do
+      expect(subject.respond_to?(:<<)).to be true
```

Comment:
`expect(subject).to respond_to(:<<)`

Maybe?

Reply:
Didn't know that was an rspec thing.  TIL.

Can fix.